packages/flutter/test/material/menu_anchor_test.dart

```diff
@@ -4947,6 +4947,91 @@ void main() {
       expect(find.byIcon(disabledIcon), findsOneWidget);
     });
   });
+
+  testWidgets('MenuAnchor.of can return the nearest MenuController', (WidgetTester tester) async {
+    final MenuController controller = MenuController();
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Scaffold(
+          body: Center(
+            child: MenuAnchor(
+              controller: controller,
+              menuChildren: const <Widget>[MenuItemButton(child: Text('menu item'))],
+              builder: (BuildContext context, MenuController controller, Widget? child) {
+                return FilledButton(
+                  onPressed: () {
```

Comment:
Technically this FilledButton can use a `onPressed: () {}` and `Text('Button')` to make the test shorter.